src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -92,23 +109,24 @@ class ServiceTypeAhead extends Component {
               {isOpen && inputValue.length >= 2 ? (
                 <div className="dropdown" role="listbox">
                   {services
-                    .filter(svc => this.shouldShow(inputValue, svc))
-                    .map((svc, index) => (
+                    .filter(specialty => this.shouldShow(inputValue, specialty))
+                    .map((specialty, index) => (
                       <div
-                        key={svc.name}
+                        key={this.getSpecialtyName(specialty)}
                         {...getItemProps({
-                          item: svc,
+                          item: specialty,
                           className: this.optionClasses(
                             index === highlightedIndex,
                           ),
                           role: 'option',
                           'aria-selected': index === highlightedIndex,
                         })}
                         style={{
-                          fontWeight: selectedItem === svc ? 'bold' : 'normal',
+                          fontWeight:
```

Comment:
Is it possible to use system design classes instead?

Reply:
Actually that code wasn't doing anything, so I'm deleting it.